client/tests/integration/asset.rs

```diff
@@ -225,7 +225,6 @@ fn client_add_asset_with_name_length_more_than_limit_should_not_commit_transacti
         .into_iter()
         .map(|asset| asset.id().clone())
         .collect::<Vec<_>>();
-    dbg!(&asset_definition_ids);
```

Comment:
Not strictly necessary. Consider turning this into a `iroha_logger::debug!` invocation.

Reply:
Adding other debug invocations to help debugging also helps.